controllers/get-articles.controller.js

```diff
@@ -0,0 +1,18 @@
+const getArticlesIdModule = require('../models/get-articles.model')
+
+function getArticlesId(request, response, next) {
+    const { article_id } = request.params
+    if (isNaN(Number(article_id))) {
```

Comment:
Your right that [isNaN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/isNaN#:~:text=This%20behavior%20of%20isNaN()%20for%20non%2Dnumeric%20arguments%20can%20be%20confusing!) on its own could give some unexpected behaviour here, not least because article_id will be a string from the parameterised endpoint url so I see why you've combined it with Number.isNaN( ) but...
Remember that the database is expecting the column value of article_id to be a number so will give a specific error in the case of article_id being something else.

Look at the error that comes back from the database if you pass anything other than a string-integer to query. You could handle what is returned with a custom error handling middleware in app.js

Reply:
Thank you!